server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/DataSourceController.java

```diff
@@ -178,6 +182,14 @@ public SuccessResponse<ConnectionPreviewBatchImportResp> previewBatchImportDataS
         return Responses.success(connectionBatchImportPreviewer.preview(file));
     }
 
+    @ApiOperation(value = "downloadDatasourceTemplate", notes = "Download Datasource Template File")
+    @RequestMapping(value = "/datasources/template", method = RequestMethod.GET)
```

Comment:
may POST  downloadBatchImportTemplate

Reply:
btw, the other download apis are all 'GET' method~~